src/test/java/uk/gov/companieshouse/overseasentitiesapi/interceptor/UserAuthenticationInterceptorTest.java

```diff
@@ -87,4 +183,17 @@ void testInterceptorReturnsTrueWhenAnApiKeyIsUsed() {
         assertTrue(result);
         assertEquals(HttpServletResponse.SC_OK,  mockHttpServletResponse.getStatus());
     }
+
+    private void ericTokenSet(boolean companyExists) {
+        String tokenValue = null;
+        if (companyExists) {
+            tokenValue = "company_number=" + VALID_OE_NUMBER;
+        }
+        setStubbing(tokenValue);
+    }
+    private void setStubbing(String companyNumberToken) {
```

Comment:
Minor: A blank line here would improve readability. There are couple of double of blank lines above too.